File: src/Reference.VB/GeoEntity.vb
Imports System.ComponentModel.DataAnnotations

''' <summary>
''' Geo Entity
''' </summary>
Public Class GeoEntity
    ''' <summary>
    ''' UUID
    ''' </summary>
    <Key>
    Public Property Uuid As Guid
    ''' <summary>
    ''' Latitude
    ''' </summary>
    Public Property Latitude As Double
    ''' <summary>
    ''' Longitude
    ''' </summary>
    Public Property Longitude As Double
    ''' <summary>
    ''' Place Id
    ''' </summary>
    Public Property PlaceId As Long
    ''' <summary>
    ''' Display name
    ''' </summary>
    Public Property DisplayName As String
    ''' <summary>
    ''' City name
    ''' </summary>
    Public Property City As String
    ''' <summary>
    ''' County
    ''' </summary>
    Public Property County As String
    ''' <summary>
    ''' State
    ''' </summary>
    Public Property State As String
    ''' <summary>
    ''' Country
    ''' </summary>
    Public Property Country As String
    ''' <summary>
    ''' Country code
    ''' </summary>
    Public Property CountryCode As String
End Class

File: src/Reference.CSharp/GeoEntity.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Reference.CSharp
{
    /// <summary>
    /// Geo Entity
    /// </summary>
    public class GeoEntity
    {
        /// <summary>
        /// UUID
        /// </summary>
        [Key]
        public Guid Uuid { get; set; }
        /// <summary>
        /// Latitude
        /// </summary>
        public double Latitude { get; set; }
        /// <summary>
        /// Longitude
        /// </summary>
        public double Longitude { get; set; }
        /// <summary>
        /// Place Id
        /// </summary>
        public long PlaceId { get; set; }
        /// <summary>
        /// Display name
        /// </summary>
        public string DisplayName { get; set; }
        /// <summary>
        /// City name
        /// </summary>
        public string City { get; set; }
        /// <summary>
        /// County
        /// </summary>
        public string County { get; set; }
        /// <summary>
        /// State
        /// </summary>
        public string State { get; set; }
        /// <summary>
        /// Country
        /// </summary>
        public string Country { get; set; }
        /// <summary>
        /// Country code
        /// </summary>
        public string CountryCode { get; set; }
    }
}